boards/xtensa/esp32/esp32-devkitc/README.txt

```diff
@@ -39,6 +39,9 @@ STATUS
   Espressif is working to include support to WiFi and Bluetooth, but
   it will depends on their external libraries to get it up and running.
 
+  Wifi is supported and there is a config to demonstrate its use.  Please, see
+  configs/wapi.
```

Comment:
how can i prepare `/mnt/esp/wifi`?

Reply:
@yamt There is a config for that: ESP32_WIFI_FS_MOUNPT, it has the default value of /mnt/esp/wifi.  The board bringup mounts that.

Also note that you can disable saving WIFI parameters.